### ISSUE TITLE
Write Provided ServiceAccount Documentation

### ISSUE THREAD

**Issue by everettraven** [operator-framework/operator-controller#977]
# User story

Describe the user story or stories for this epic:

- As a user, I expect to be able to define a ServiceAccount that OLM v1 uses to manage content
- As a user, I expect the ClusterExtension installation to fail if the provided ServiceAccount has insufficient permissions
- As a user, I expect the ClusterExtension to be rejected if a ServiceAccount is not provided
- As a user, I expect to be able to identify the permissions required to install a bundle so that I can create a ServiceAccount with all the permissions necessary to install an extension
     
## List the user-facing issues in the epic that require documentation.
- #971 

## Does this epic require new content or updates to existing content?

- [x] Net-new content
- [x] Update existing content

> [!NOTE]
> Preemptively setting that this will need both. I'm not 100% sure, but I think it is likely that both existing content will need to be updated and that new content will need to be written

## What type of documentation is required for this epic?

- [x] Release note
- [x] Concept
- [ ] Procedure
- [x] Reference

## Is there prior art or reference materials that would help write the documentation?
- Once the feature is complete and in place, there will likely be some unit and e2e tests that could be used as reference

Documentation should be contributed to the [`docs/drafts/` directory](https://github.com/operator-framework/operator-controller/tree/main/docs/drafts)

**Comment by trgeiger** [comment]
I think this might effectively be a duplicate of #1130 ? The bulk of the information that would be in the docs for this ticket 977 would be the derivation of required permissions so I'm not sure if it's necessary to separate these 2 issues?

**Comment by LalatenduMohanty** [comment]
PR under https://github.com/operator-framework/operator-controller/pull/1232/
